messaging.rst

```diff
@@ -60,7 +60,7 @@ Invariants
 - Expiration must be larger than the current block number and smaller than the channel’s settlement period.
 
 Hash
-^^^^
```

Comment:
Why did you change this?

Reply:
My thing complained locally. Even though it does not seem to be smaller.